src/main/java/duckie/exception/DuckieException.java

```diff
@@ -0,0 +1,23 @@
+package duckie.exception;
+
+/**
+ * Parent class of all the possible DuckieExceptions
+ */
+public class DuckieException extends Exception {
+    /**
+     * Instantiate DuckieException
+     * @param message Error message of why the Exception is thrown
+     */
+    public DuckieException(String message) {
+        super(message);
+    }
+
+    /**
+     * Overrides method to show the message of a DuckieException
+     * @return
+     */
+    @Override
+    public String toString() {
+        return getMessage();
+    }
```

Comment:
There might not really be a need to override this method since you won't be really printing out the exception object created.